### PR TITLE
Label component inspector localization

### PR DESCRIPTION
**Pull request by shrinktofit**
Re: #

### Changelog

*

image

image

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by LinYunMo — APPROVED** [review]
Good work

File: editor/i18n/zh/modules/ui.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                            '1. CLAMP: 节点约束框之外的文字会被截断 <br> ' +
+                            '2. SHRINK: 自动根据节点约束框缩小文字<br> ' +
+                            '3. RESIZE: 根据文本内容自动更新节点的 height 属性.',
+                    },
+                    'enableWrapText': {
+                        displayName: '自动换行',
+                        tooltip: '自动换行。',
+                    },
+                    'useSystemFont': {
+                        displayName: '使用系统字体',
```
Comment (jareguo) [inline editor/i18n/zh/modules/ui.js]: 感觉不需要“使用”二字，因为这是一个 checkbox 控件。就好像自动换行，也没有叫做“使用自动换行”

File: editor/i18n/zh/modules/ui.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        tooltip: '是否使用系统默认字体，选中此项会将引用的字体资产置空。',
+                    },
+                    'fontFamily': {
```
Comment (jareguo) [inline editor/i18n/zh/modules/ui.js:60]: 字体族吧？比较符合 CSS 教程

File: editor/i18n/zh/modules/ui.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                        displayName: '字体集',
```
Comment (shrinktofit) [inline editor/i18n/zh/modules/ui.js]:
```suggestion
                        displayName: '字体族',
```

File: editor/i18n/zh/modules/ui.js (hunk 2 above)
Comment (shrinktofit) [reply editor/i18n/zh/modules/ui.js:60]: https://github.com/cocos/cocos-engine/pull/16234/commits/7ec127f75dfa6cb29654123de8c63fab2de81550

File: editor/i18n/zh/modules/ui.js (hunk 1 above)
Comment (shrinktofit) [reply editor/i18n/zh/modules/ui.js]: Sovled.

**Review by jareguo — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**